packages/tomls/src/core/configs/feature_flags.toml

```diff
@@ -184,16 +184,7 @@ depends = ["invoke.upgrade_core_proxy"]
 target = ["CoreProxy"]
 fromCall.func = "owner"
 func = "setAdmins"
-args = [
-    "<%= keccak256(hexlify('camelotSwapPublisher')) %>",
-    [
-        "<%= settings.camelotSwapPublisherAdmin1 %>",
-        "<%= settings.camelotSwapPublisherAdmin2 %>",
-        "<%= settings.camelotSwapPublisherAdmin3 %>",
-        "<%= settings.camelotSwapPublisherAdmin4 %>",
-        "<%= settings.camelotSwapPublisherAdmin5 %>",
-    ],
-]
+args = ["<%= keccak256(hexlify('camelotSwapPublisher')) %>", []]
```

Comment:
reset all admins for "camelotSwapPublisher" feature flag - this feature is not used anymore